README.en-US.md

```diff
@@ -33,8 +33,9 @@ Download this project, install dependencies and start the project.
 $ git clone https://github.com/imgcook/imove.git
 $ npm run example
 ```
-
+<!-- markdown-link-check-disable -->
```

Comment:
refer to https://github.com/gaurav-nelson/github-action-markdown-link-check#disable-check-for-some-links